packages/nimble-components/src/utilities/number-formatter/unit-scale/models/tests/manually-translated-unit-scale.spec.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable max-classes-per-file */
+import { ManuallyTranslatedUnitScale } from '../manually-translated-unit-scale';
+import { UnitPrefix } from '../unit-prefix';
+import { UnitTranslation } from '../unit-translation';
+
+describe('ManuallyTranslatedUnitScale', () => {
+    class TestManuallyTranslatedUnitScale extends ManuallyTranslatedUnitScale {
+        protected override getUnitTranslations(): Map<string, UnitTranslation> {
+            return new Map<string, UnitTranslation>([
+                ['en', new UnitTranslation('byte', 'bytes', 'B')]
+            ]);
+        }
+
+        protected override getSupportedPrefixes(): UnitPrefix[] {
+            return [new UnitPrefix(1000, 'k'), new UnitPrefix(1000000, 'M')];
+        }
+    }
+
+    const unitScale = new TestManuallyTranslatedUnitScale();
+
+    it('automatically includes base unit', () => {
+        expect(unitScale.baseScaledUnit).toBeDefined();
+    });
+
+    it('creates expected units from prefixes', () => {
+        const units = unitScale.supportedScaledUnits.sort((a, b) => (a.scaleFactor < b.scaleFactor ? -1 : 1));
```

Comment:
`unitScale.supportedScaledUnits` is already sorted, right? Is there a reason you need to call `sort` here?

Reply:
Being sorted is an implementation detail. Only `UnitScale` itself knows that/relies on that. So if my test needs it sorted, it has to sort it.